docs/proposals/20200804-windows-support.md

```diff
@@ -0,0 +1,286 @@
+---
+title: Windows kubeadm-based worker nodes support
+authors:
+  - "@jsturtevant"
+  - "@ksubrmnn"
+reviewers:
+  - "@CecileRobertMichon"
+  - "@ncdc"
+  - "@randomvariable"
+creation-date: 2020-08-25
+last-updated: 2020-09-09
+status: implementable
+see-also:
+---
+
+# Windows kubeadm-based worker nodes support
+
+## Table of Contents
+
+- [Windows kubeadm-based worker nodes support](#windows-kubeadm-based-worker-nodes-support)
+  - [Table of Contents](#table-of-contents)
+  - [Glossary](#glossary)
+  - [Summary](#summary)
+  - [Motivation](#motivation)
+    - [Goals](#goals)
+    - [Non-Goals/Future Work](#non-goalsfuture-work)
+  - [Proposal](#proposal)
+    - [Cluster API Bootstrap Provider Kubeadm](#cluster-api-bootstrap-provider-kubeadm)
+      - [cloud-init and cloudbase-init](#cloud-init-and-cloudbase-init)
+      - [Image Creation](#image-creation)
+      - [Kubelet and other component configuration](#kubelet-and-other-component-configuration)
+      - [netbios names](#netbios-names)
+    - [Infrastructure provider implementation](#infrastructure-provider-implementation)
+    - [User Stories](#user-stories)
+      - [As an operator, I would like to create Windows OS worker nodes with the CAPI API.](#as-an-operator-i-would-like-to-create-windows-os-worker-nodes-with-the-capi-api)
+      - [As an operator, I would like to manage Windows OS worker nodes with the CAPI API.](#as-an-operator-i-would-like-to-manage-windows-os-worker-nodes-with-the-capi-api)
```

Comment:
Seems like some type active directory awareness would be worth diving into.  
```
- [As an operator, i'd like to federate my windows OS worker nodes with active directory]
```
or is that out of scope?  Just a thought from what we think our customers would like to be able to do

Reply:
(feel free to merge if its ready , dont wanna hold things up, maybe can iterate if this is in scope)

Reply:
Yes definitely. I think we decided to move forward with this and address the AD via https://github.com/kubernetes-sigs/cluster-api/issues/3761

ref: https://github.com/kubernetes-sigs/cluster-api/pull/3616#issuecomment-708358305